warehouse/admin/templates/admin/base.html

```diff
@@ -131,7 +131,7 @@
           </a>
         </li>
         <li>
-          <a href="{{ request.route_path('admin.verdicts.list') }}">
+          <a href="{{ request.route_path('admin.verdicts.list') }}?classification=threat&manually_reviewed=0">
```

Comment:
cc @ewdurbin: I _think_ we can pass these default parameters via `route_path`, e.g.,

```python
request.route_path('admin.verdicts.list', classification="threat", manually_reviewed=0)
```

but I'm not positive.

Reply:
Checks out: https://docs.pylonsproject.org/projects/pyramid/en/latest/api/request.html#pyramid.request.Request.route_path